src/current/v23.2/cluster-virtualization-overview.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Cluster Virtualization Overview
+summary: Learn more about cluster virtualization and how it facilities cluster-to-cluster replication.
+toc: true
+docs_area: deploy
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+
+Refer to the [Known Limitations](#known-limitations) section for further detail.
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} This page gives an overview of _cluster virtualization_ in CockroachDB {{page.version.version}}. Cluster virtualization allows you to separate a cluster's _control plane_ from its _data plane_. The cluster's control plane manages cluster nodes and node-to-node traffic, while its data plane reads data from and writes data to the cluster's storage.
+
+{{site.data.alerts.callout_success}}
```

Comment:
I would drop this tip box to below the two bullet point explanation, so that the explanation is not interrupted.

Reply:
This really feels more like a note to me as well.

Reply:
So, I made it a tip because it's a tip that **no extra action** is needed on your part to enable CV besides enabling PCR. If action was needed, I'd make it a note. WDYT?